### DISCUSSION TITLE
[iOS bug. Moved to issues] Ability to disable link modals / popups

### DISCUSSION THREAD

**Discussion by benwoodward** [roam-garden/i#53] [Feature requests & Ideas] [answered]
I'm not a fan of the modals that appear when you press on a link on mobile. I'd prefer it if on mobile links just loaded the page they link to. I'd like to be able to disable this feature via a setting.

**Comment by Stvad** [comment, marked as answer]
I'm not quite sure I follow this one. Can you elaborate/post a screenshot of undesired behavior?

**Reply by benwoodward** [reply]
![image](https://user-images.githubusercontent.com/1472981/112788362-3eaef380-90b7-11eb-9599-21e1a31ce209.jpeg)

This the modal I'm referring to. It appears when you press a link on mobile

**Reply by Stvad** [reply]
Interesting. this seems to be a live preview window (which shows up on desktop on mouse hover). For me on mobile it only shows up if I do "long press", usual clicks just open the page.   
This seems like it may be a bug for some specific env.   

Can you please add information on the Browser/OS versions you have?

**Reply by benwoodward** [reply]
I've tested it in Safari for iOS and Chrome for iOS and it occurs in both of these.

EDIT: iOS 14.4.1

**Reply by Stvad** [reply]
I see. Let me find an iOS device to test things with 🤔.  Btw just in case: can include browser versions too?

**Reply by benwoodward** [reply]
Chrome iOS version: 87.0.4280.77
Safari iOS is 14.4.1 (it's bundled with iOS so if you're on iOS 14.4 you'll have same safari.

**Reply by Stvad** [reply]
Confirmed it on another Iphone 🙁. You can double-click to actually go to the page apparently.  Created an issue to track resolving this: https://github.com/roam-garden/i/issues/54
